app/forms/steps/solicitor_declaration_form.rb

```diff
@@ -35,8 +37,8 @@ def cost_totals(application)
         },
         {
           'cost_type' => 'disbursements',
-          'amount_with_vat' => 0,
-          'amount' => 0
+          'amount_with_vat' => CostCalculator.cost(:disbursement_total, application, true),
+          'amount' => CostCalculator.cost(:disbursement_total, application, false)
```

Comment:
as a side note for a future refactoring it make more sense to change the CostCalculator interface so that:
```
amount_without_vat, amount_with_vat = CostCalculator.cost(:disbursement_total, application)
```
as this helps avoid having to run the same loops twice.

Thinking about doing something similar in CW to make that nicer as well...

Reply:
agreed, will do this when this is all finalised in the core costs ticket